api/api_models/skills.py

```diff
@@ -0,0 +1,60 @@
+from pydantic import BaseModel, Field, EmailStr
+from datetime import datetime
+from typing import Optional
+
+
+class SkillBase(BaseModel):
+    name: str
+
+    class Config:
+        orm_mode = True
+
+
+class UserBase(BaseModel):
+    pass
+
+    class Config:
+        orm_mode = True
+
+
+class SkillCreate(SkillBase):
+    name: str
+    
+
+
+class Skills(BaseModel):
+    id: int
+    name: str
+    user_id: int
+
+    class Config:
+        orm_mode = True
+
```

Comment:
I don't quite get how these "skill" related models work together. Can you clean these up? @eeTiboah can you assist?